src/assets/thirdPartyServices/mozi/gene_annotation_service/form/index.js

```diff
@@ -291,7 +295,11 @@ const AnnotationForm = props => {
           <FormGroup row>
             <FormControlLabel
               control={<Checkbox color="primary" onChange={e => toggleAnnotation("gene-go-annotation", e)} />}
-              label="Gene-GO"
+              label={
+                <a href="http://www.geneontology.org" target="_blank">
```

Comment:
use the `AnchorLink` component from our common directory `src/components/common/AnchorLink.js`. 
you just have to pass `newTab` as a prop, if you want to open it in new tab, it will take care of the rest.